pyproject.toml

```diff
@@ -61,3 +61,38 @@ version_file = "_icepyx_version.py"
 version_file_template = 'version = "{version}"'
 local_scheme = "node-and-date"
 fallback_version = "unknown"
+
+
+[tool.ruff.lint]
+select = [
+  "E",
+  "F",
+]
+ignore = [
+  # Line too long
+  # NOTE: This is a formatting concern. Formatter handles long lines of code,
+  # but allows inline comments to be infinitely long (automatically formatting
+  # them can have unintended consequences). In our codebase, we have a lot of
+  # overlong comments.
+  # See: https://github.com/psf/black/issues/1713#issuecomment-1357045092
+  "E501",
+  # TODO: remove ignores below this line
+  # comparison syntax in tests
+  "E721",
+  # bare except
+  "E722",
+  # unable to detect undefined names
+  "F403",
```

Comment:
Some of these are pretty easy to polish off. F403 is one -- here we import `*` in an `__init__.py`. Instead, we should import the objects we want to expose by name. Any thoughts on what objects we want to expose here?

https://github.com/icesat2py/icepyx/blob/development/icepyx/quest/dataset_scripts/__init__.py